prettier/prettier-check.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: prettier-check-from-gitrepo
```

Comment:
This Task doesn't require a git repo does it?  It looks like it'll work on any code provided in a workspace.  Suggest `prettier-check` as the Task name if that's the case, dropping the "from-gitrepo" bit.

Reply:
I do require a git repo. I notice that there is already a git-clone task in the catalog, which will fetch the code from the git repo. Should I add prettier-check task which acts the same as git-clone?

Reply:
The cool thing about workspaces is that you don't need to know if it's a git repo or not! :D 
You could also have a workspace that's populated by a GCS bucket, a hg/svn repo, or anything else!

All this task needs to care about is that some filesystem is being passed to it.